tools/syz-testmodules/testmodules.go

```diff
@@ -0,0 +1,138 @@
+// Copyright 2023 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+// syz-testmodules tests the translation of coverage addresses between fuzzer instances with differing module offsets.
```

Comment:
We normally use `tools/*` for syzbot maintenance / working with bugs, but not for testing syzkaller code. The standard place for tests are `*_test.go` files (preferably located right in the package they're testing), this way it's also ensured that we can easily run all tests we have with just `go test ./...`  and don't miss anything.

Reply:
Yes, please move tests to _test.go file in the package.
But I like how the syz-manager changes look now!

Reply:
Sounds good! I've moved this file to pkg/cover/canonicalizer_test.go